client/components/CheckoutCartSummCard.js

```diff
@@ -0,0 +1,18 @@
+import React, {Fragment} from 'react'
+
+const CheckoutCartSummCard = ({name, price, orderItem, totalItem}) => {
+  const numberWithCommas = num => {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  }
+  return (
+    <Fragment>
+      <td>{name}</td>
+      <td className="right aligned">{`$${numberWithCommas(price)}`}</td>
+      <td className="right aligned">x</td>
+      <td className="right aligned">{numberWithCommas(orderItem.quantity)}</td>
+      <td className="right aligned">{`$${numberWithCommas(totalItem)}`}</td>
```

Comment:
Use
 .toLocaleString(undefined, {style: 'currency', currency: 'USD'})
instead of creating numberWithCommas function

Reply:
see above